gnucash/gnome/search-owner.c

```diff
@@ -73,33 +73,7 @@ enum
 static guint signals[LAST_SIGNAL] = { 0 };
 #endif
 
-GType
-gnc_search_owner_get_type (void)
-{
-    static GType type = 0;
-
-    if (!type)
-    {
-        GTypeInfo type_info =
-        {
-            sizeof(GNCSearchOwnerClass),    /* class_size */
-            NULL,                           /* base_init */
-            NULL,                           /* base_finalize */
-            (GClassInitFunc)gnc_search_owner_class_init,
-            NULL,                           /* class_finalize */
-            NULL,                           /* class_data */
-            sizeof(GNCSearchOwner),         /* */
-            0,                              /* n_preallocs */
-            (GInstanceInitFunc)gnc_search_owner_init,
-        };
-
-        type = g_type_register_static (GNC_TYPE_SEARCH_CORE_TYPE,
-                                       "GNCSearchOwner",
-                                       &type_info, 0);
-    }
-
-    return type;
-}
```

Comment:
This compiles and gnucash runs but I'm not really sure which dialogue this is about. So I haven't tested that yet.

Reply:
It's the find customer/vendor/employee dialog. Internally those are all subclasses of owner.

You know that you can run ```gcc -E``` on a file to see what actual C code macros emit, right?

Reply:
No, I didn't. I was just trying to find out how I could get that output. Thanks!